dht.go

```diff
@@ -365,10 +365,15 @@ func makeRtRefreshManager(dht *IpfsDHT, cfg dhtcfg.Config, maxLastSuccessfulOutb
 		return err
 	}
 
+	pingFnc := func(ctx context.Context, p peer.ID) error {
+		return dht.protoMessenger.Ping(ctx, p)
```

Comment:
> I'm using the `ping` package to probe the remote peer instead of the ping message from the `/ipfs/kad/1.0.0` protocol

Not deeply familiar with the codebase. Just double checking, is this really not using the Kademlia Ping mechanism?

Reply:
After yesterdays discussion, we changed it back to the Kademlia PING message. So this is indeed the Kademlia Ping.

Reply:
(although perhaps used in an older version? did not check)